web/src/utils/state/atoms.ts

```diff
@@ -51,3 +51,7 @@ export const dataSourcesCollapsedBarBreakdown = atom(true);
 export const dataSourcesCollapsedBreakdown = atom(true);
 
 export const dataSourcesCollapsedEmission = atom(true);
+
+export const userLocationAtom = atom(undefined);
```

Comment:
Do we actually need to keep this in an atom? What are we going to use it for outside the survey component? :)